pages/accountLists/[accountListId]/tools/fixPhoneNumbers.page.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { ReactElement } from 'react';
+import { useTranslation } from 'react-i18next';
+import Head from 'next/head';
+import FixPhoneNumbers from '../../../../src/components/Tool/FixPhoneNumbers/FixPhoneNumbers';
+
+const fixPhoneNumbers = (): ReactElement => {
```

Comment:
React components are usually capitalized
```suggestion
const FixPhoneNumbers = (): ReactElement => {
```

I kinda like just exporting these directly as named exports. With default exports, it's another line at the bottom and then since the import isn't named, you can name it wrong or introduce a typo.

Reply:
I tried doing named exports but it wouldnlt load properly. I looked into the error and it looks to be nextjs related (couldnt find a solution but didnt extensivley check) if we want change all of pages to named exports we can do that seperately I guess

Reply:
I forgot this was a page when I commented... Ya I think Next.js needs default exports for pages. You're good :)